editor/components/TableWrapper.tsx

```diff
@@ -1,25 +1,27 @@
-import { useState, useEffect, useMemo } from "react";
```

Comment:
sorry for this import diff i was sorting all the imports bc they were getting long and messy :sweat_smile:

Reply:
much better ☺️